src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './Navbar.scss';
+
+export const Navbar: React.FC = () => (
+  <nav className="nav">
+    <div className="logo">
+      <Link to="/">buk klab</Link>
+    </div>
+    <ul>
+      <li>
+        <Link to="/books">books</Link>
+      </li>
+      <li>
+        <Link to="/members">members</Link>
+      </li>
+      <li>
+        <Link to="/about">about</Link>
+      </li>
+      <li>
+        <Link to="/signin">sign in</Link>
+      </li>
```

Comment:
Keep it for now like this. If you will start adding more, we might want to go with list of links with labels and iterating over them